src/pendulum/value_iteration.py

```diff
@@ -136,6 +136,6 @@ def _DoCalcVectorOutput(self, context, input, state, output):
 
 # Visualize the result as a video.
 vis = PendulumVisualizer()
-ani = vis.animate(logger, 60, repeat=True)
+ani = vis.animate(logger, resample=False, repeat=True)
```

Comment:
Here, you're relying on PendulumVisualizer's default draw rate to match the log rate for the SignalLogger... PendulumVisualizer defaults to 60hz, but the logger is set with timesteps of 0.033333?

It's seems a little weird to talk about rates for the Visualizer but sampling timesteps for the visualizer -- but I don't think it breaks anything (as either resample=True -> the desired rate is achieved by interpolation, and numerical mismatches from translating rate <-> timestamp don't matter, or resample=False, and rate only affects how fast the animation plays, so numerical error is also fine).

Reply:
good call.  changed everything to timestep.